func-consumption/relauncher.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
what is the purpose of this?

would be good to add a header comment to explain

Reply:
This is save me from Microsofts decision to just let the `func` process exit every time it gets a network timeout.  By using this handy-dandy `relaucnher` script every time the process dies it will get relaunched.

I use it for long running end to end tests. 

Will add a header as suggested. 👍